src/components/Palette/Palette.js

```diff
@@ -1,15 +1,36 @@
 import React, { Component } from 'react';
 import { Color } from '../Color/Color';
+import { fetchApiData } from '../../utils/api';
 
 export class Palette extends Component {
   constructor() {
     super();
-    this.state = { colors: [] };
+    this.state = { 
+      colors: [],
+      savedProjects: [],
+      savedPalettes: [],
+    };
   }
 
-  componentDidMount = () => {
+  componentDidMount = async () => {
     this.regeneratePalette();
     document.addEventListener('keydown', this.handleKeydown);
+    this.fetchProjects()
+    this.fetchPalettes()
+  }
+
+  fetchProjects = async () => {
+    let projects = await fetchApiData('projects')
+    this.setState({
+      savedProjects: projects[0]
```

Comment:
Setting state with first item in array. Can probably spread it in with more data.